cli/cmd/brokers.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(brokersCmd)
+	brokersCmd.AddCommand(createCmd)
+	brokersCmd.AddCommand(describeCmd)
+	brokersCmd.AddCommand(listCmd)
+	brokersCmd.AddCommand(deleteCmd)
+}
+
+var brokersCmd = &cobra.Command{
+	Use:   "brokers",
+	Short: "manage brokers associated with a service controller",
+	Long:  "Manage brokers associated with a service controller",
+}
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Add a service broker to service controller",
+	Long:  "Add a service broker to service controller",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return fmt.Errorf("Not implemented yet")
+	},
+}
+
+var describeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "Describe the specified service broker",
+	Long:  "Describe the specified service broker",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return fmt.Errorf("need name of the broker")
```

Comment:
I guess if we could make it user-friendly, and not just a simpler way to call the REST API, then we should, so I think the create would be like cf's:
```
sc broker create SB_NAME USER PWD URL
```
so I take it back, it should be name instead of ID.  If this doesn't workout we can change it later.